pom.xml

```diff
@@ -26,6 +26,8 @@
         <java.version>1.8</java.version>
         <maven.compiler.source>${java.version}</maven.compiler.source>
         <maven.compiler.target>${java.version}</maven.compiler.target>
+        <maven.javadoc.plugin.version>3.2.0</maven.javadoc.plugin.version>
```

Comment:
Both versions are provided by parent. Please use the ones from the parent and do not override them

Reply:
Fixed